src/components/Comments/GetComment/AllComments.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { FunctionComponent } from "react";
+import { Box, Button, Typography, makeStyles } from "@material-ui/core";
+import TabViewIntroSection from "../../TabViewIntroSection";
+
+const getAllCommentEndpoint = ["GET /comments"];
+const getAllCommentHeading = "GET all comments";
+const getAllCommentSubtitle = "Need to get some comments?";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.background.paper,
+  },
+  buttonGroup: {
+    maxWidth: 800,
+  },
+  button: {
+    marginLeft: 10,
+    marginRight: 10,
+  },
+}));
+
+const AllComments: FunctionComponent = () => {
+  const classes = useStyles();
+
+  const handleFlagFilter = (event: any) => {};
+
+  const handleRefIdFilter = (event: any) => {};
+
+  const handleOwnerIdFilter = (event: any) => {};
+
+  const handleOriginFilter = (event: any) => {};
+
+  return (
+    <Box mt={10} mb={5}>
```

Comment:
This is problematic. You probably tried to vertically center the content of the tab, as seen in the figma design, with a margin-top of 10 * 8 = 80px and a margin-bottom of 5 * 8 = 40px. You should never do this because what if the device of the screen is not the same size as the one you are developing with? This doesn't scale. Don't use absolute values, you need to use relative values.